pushover/README.md

```diff
@@ -0,0 +1,85 @@
+# Pushover  
+
+Pushover is one out of many push services, which is compatible with Android, IOS and Windows Clients. This plugin make it possible to push messages to your clients from smarthomeNG.
+  
+## Requirements
+  
+### Python libraries  
+* requests - [install instructions](http://docs.python-requests.org/en/latest/user/install/#install "http://docs.python-requests.org/en/latest/user/install/#install")
```

Comment:
I would list it if we list them in other plugins too. On the other hand one can check the requirements file to know which packages are required.